extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -34,6 +36,10 @@ require([
 		amazon.call();
 	}
 
+	if (geo.isProperGeo(instantGlobals.wgAdDriverOpenXSpeedBidderCountries)) {
+		speedBidder.call();
+	}
+
 	mercuryListener.onLoad(function () {
 		if (geo.isProperGeo(instantGlobals.wgAdDriverOpenXBidderCountriesMobile)) {
```

Comment:
I wonder if we shouldn't disable regular OpenXBidder when `geo.isProperGeo(instantGlobals.wgAdDriverOpenXSpeedBidderCountries)`